README.md

```diff
@@ -42,7 +42,7 @@ UNENCRYPTED_SECRET="$(vault -y $MY_ENCRYPTED_SECRET)"
 
 Obviously you need to make sure that in the context of running the vault,
 there is some sort of way for providing kms permissions by for example adding the decryptPolicy managed policy
-from the vault cloudformation stack to the ec2 instance or whatever runs the code.
+from the vault Cloudformation stack to the ec2 instance or whatever runs the code.
```

Comment:
If we are to be accurate with all the terms, Cloudformation should be CloudFormation, "kms" should be "KMS" and "ec2" should be "EC2" etc. IMHO, using these in this context with all lowercase is fine as the reader most likely understands what we are referring to.